src/Jobs/ChannelOpenJob.cs

```diff
@@ -68,9 +70,49 @@ public async Task Execute(IJobExecutionContext context)
             });
 
             _logger.LogError(e, "Error on {JobName}", nameof(ChannelOpenJob));
+            if (!shouldRetry)
+            {
+                await context.Scheduler.DeleteJob(context.JobDetail.Key, context.CancellationToken);
+            }
```

Comment:
What if we implement a retry feature? wouldn't this kill that possibility?

Reply:
This only kills the retry for the specific case that the channel rejects funding. Because we know there's no point in retrying. Every other error still goes through the retry intervals 1,2,5,10,20